### PR TITLE
[DOCS] Less verbose logging during docs build

### PR DESCRIPTION
**Pull request by anthonyburdi**
Default to less verbose logging during docs build. Parameters added to use more verbose output during debugging.

### CODE REVIEW

File: docs/docusaurus/scripts/remark-named-snippets/snippet.js [hunk 1]
```diff
@@ -191,12 +191,13 @@ function main() {
     delete snippet.file // Remove duplicate filename to clean up stdout
     out[file].push(snippet)
   }
-  console.log(out)
+  if (verbose) {
+    console.log(out)
+  }
```
Comment (Shinnnyshinshin) [inline docs/docusaurus/scripts/remark-named-snippets/snippet.js:209]: ❤️

File: docs/docusaurus/scripts/remark-named-snippets/snippet.js [hunk 2]
```diff
@@ -173,7 +173,7 @@ function getDirs() {
   return manualDirs.concat(versionDirs)
 }
 
-function main() {
+function main(verbose = false) {
```
Comment (Shinnnyshinshin) [inline docs/docusaurus/scripts/remark-named-snippets/snippet.js:187]: Where would we (if ever) run this function as `verbose = true`?

Comment (Shinnnyshinshin) [reply docs/docusaurus/scripts/remark-named-snippets/snippet.js:187]: If so, how would we do this?

Comment (anthonyburdi) [reply docs/docusaurus/scripts/remark-named-snippets/snippet.js:187]: Good call, in https://github.com/great-expectations/great_expectations/pull/7894/commits/19b2b1f9a76f64e6182196b5fe293a0cb168c9c6 I changed `yarn snippet-check` to use the verbose mode which you can run locally to check all snippets.

**Review by alexsherstinsky — APPROVED** [review]
LGTM -- thank you!

**Pull request merged**